search-enterprise/latest/config/transactional.md

```diff
@@ -0,0 +1,382 @@
+* Using the database gives transactional consistency as opposed to the eventual consistency provided by Elasticsearch.
+* If you use the transactional metadata query with the noindex subsystem, the search functionality in Alfresco Share won't work as it relies on full text search.
+* Normally, a query will be executed against the database, if possible. Database execution of a query depends on the query itself. It also depends on the application of an optional patch to the database, which creates the required supporting database indexes. If the supporting indexes have been created, each index subsystem can be configured to:
+* perform transactional execution of queries;
+* execute queries transactionally, when possible, and fall back to eventual consistency; or
+* always execute eventual consistency.
+* When queries are executed against the database:
+* Hidden nodes will be returned by the database, as they are in Alfresco Content Services 5.0.
+* Large result sets are not supported because Alfresco Content Services does not evaluate permissions in query but as a post filter.
+* Counts will not reflect the number of nodes that match the query.
+* The `SearchParameters` and `QueryOptions` objects can be used to override this behaviour per query.
+Alfresco Content Services supports the execution of a subset of the CMIS Query Language (CMIS QL) and Alfresco Full Text Search (AFTS) queries directly against the database. Also, the noindex subsystem supports queries only against the database. This collection of features is called transactional metadata query (TMDQ).
+
+## Options supported by Query Languages
+
+Use this information to know what options are supported by the Public API, CMIS Query Language (QL), and Alfresco Full Text Search Query Language (FTS QL).
+
+### Public API and TMDQ
+
+From public API, anything that is not a simple query, a filter query, an option that affects these, or an option that affects what is returned for each node in the results, is not supported by TMDQ.
+
+TMDQ supports:
+
+* `query`
+* `paging`
+* `include`
+* `includeRequest`
+* `fields`
+* `sort`
+* `defaults`
+* `filterQueries`
+* `scope` (single)
+* `limits` for permission evaluation
+
+The default limits for permission evaluation will restrict the results returned from TMDQ based on both the number of results processed and time taken. These can be increased, if required.
+
+The Public API does not support TMDQ for:
+
+* `templates`
+* `localisation` and `timezone`
+* `facetQueries`
+* `facetFields`
+* `facetIntervals`
+* `pivots`
+* `stats`
+* `spellcheck`
+* `highlight`
+* `ranges facets`
+
+Some of these will be ignored and produce transactional results; others will fail and be eventual.
+
+The Public API ignores the SQL select part of a CMIS query and generate the results as it would do for AFTS.
```

Comment:
```suggestion
The Public API ignores the SQL `SELECT` part of a CMIS query and generates the results as it would do for AFTS.
```

Reply:
added this in manually as Github didnt do it.

Reply:
The V1 REST API ignores the SQL `SELECT` part of a CMIS query and generates the results as it would do for AFTS.

Rechecked is there.